runtime/verbose/CMakeLists.txt

```diff
@@ -62,25 +62,52 @@ target_include_directories(j9vrb
 		${j9vm_SOURCE_DIR}/shared_common/include
 )
 
-target_link_libraries(j9vrb
-	PRIVATE
-		j9vm_interface
-		j9vm_gc_includes
-		j9vm_compiler_defines
-
-		j9stackmap
-		j9verutil
-		j9pool
-		j9modronstartup
-		j9gcvrbjava
-		j9gcvrbold
-		j9gcvrbevents
-		j9gcvrbhdlrstandardjava
-		j9gcvrbhdlrrealtime
-		j9gcvrbhdlrvlhgc
-		j9zip
-		j9utilcore
-)
+if(DEFINED OMR_OS_AIX AND DEFINED CMAKE_C_COMPILER_IS_OPENXL)
+	target_link_libraries(j9vrb
+		PRIVATE
```

Comment:
Please don't do this. If libraries are need by any compiler, they should be used for all.